src/rpdk/core/data/managed-upload-infrastructure.yaml

```diff
@@ -19,6 +27,32 @@ Resources:
             Status: Enabled
             AbortIncompleteMultipartUpload:
               DaysAfterInitiation: 1
+      VersioningConfiguration:
+        Status: Enabled
+      LoggingConfiguration:
+        DestinationBucketName: !If [CreateLogBucket, !Ref AccessLogsBucket, !Ref DestinationBucketName]
+        LogFilePrefix: ArtifactBucket
+
+  AccessLogsBucket:
+    Condition: CreateLogBucket
+    Type: AWS::S3::Bucket
+    Properties:
+      BucketName: !Ref DestinationBucketName
```

Comment:
so if we create the bucket, we set a bucket name of "" that logs to ""? That will throw some validation errors on the CloudFormation end. 

I'm all for adding an access logs bucket just don't think it is necessary for the logs bucket to log to itself.

Reply:
please check https://github.com/aws-cloudformation/cloudformation-cli/pull/396#issuecomment-598676122